iso/controller/airootfs/etc/systemd/scripts/ceph-mon-init

```diff
@@ -54,33 +82,42 @@ chown ceph /etc/ceph/ceph.mon.keyring
 
 su -s /bin/bash -c "/usr/bin/ceph-mon --cluster ceph --mkfs -i ${CHOSTNAME} --monmap /etc/ceph/monmap --keyring /etc/ceph/ceph.mon.keyring" ceph
 
-export ETCDCTL_API=3
 
 touch /var/lib/ceph/mon/ceph-${CHOSTNAME}/done
 
-
 systemctl enable ceph.target ceph-mon.target
 systemctl enable ceph-mon@${CHOSTNAME}.service
 systemctl start ceph-mon@${CHOSTNAME}.service
 
-/usr/bin/ceph auth get-or-create mgr.${CHOSTNAME} mon 'allow profile mgr' osd 'allow *' mds 'allow *' > /var/lib/ceph/mgr/ceph-${CHOSTNAME}/keyring
-systemctl enable ceph-mgr@${CHOSTNAME}.service
-systemctl start ceph-mgr@${CHOSTNAME}.service
 
-/usr/bin/ceph --cluster ceph auth get-or-create client.kube mon 'allow r, allow command "osdblacklist"' osd 'allow rwx pool=kube'
+wait_for_unit 5 ceph-mon@${CHOSTNAME}
+if [ $? -ne 0 ] ; then
+	echo "FATAL: Ceph mon did not become available after 5 seconds"
+	exit 1
+fi
 
-sleep 3
+/usr/bin/ceph --cluster ceph auth get-or-create client.kube mon 'allow r, allow command "osdblacklist"' osd 'allow rwx pool=kube'
+/usr/bin/ceph --cluster ceph auth caps client.kube mon "profile rbd" osd "profile rbd pool=kube"
+/usr/bin/ceph auth get-or-create mgr.${CHOSTNAME} mon 'allow profile mgr' osd 'allow *' mds 'allow *' > /var/lib/ceph/mgr/ceph-${CHOSTNAME}/keyring
 
-etcd_cmd () {
-	        /usr/bin/etcdctl --endpoints=http://127.0.0.1:4279  "$@"
-}
+/usr/bin/ceph config-key set mgr/prometheus/server_addr ${OPEROS_CONTROLLER_IP}
 
+systemctl enable ceph-mgr@${CHOSTNAME}.service
+systemctl start ceph-mgr@${CHOSTNAME}.service
```

Comment:
FYI: another way to enable and start at the same time is `systemctl enable --now ceph-mgr@${CHOSTNAME}.service`